gallery/src/styles/NxPageTitle/NxPageTitlePage.tsx

```diff
@@ -69,37 +66,54 @@ const NxPageTitlePage = () =>
             <td className="nx-cell"><code className="nx-code">NxPageTitle.Tags</code></td>
             <td className="nx-cell">Element</td>
             <td className="nx-cell">
-              Any "tags" to place in the page header, such as Policy Violation Indicators, should go in
-              this element.
+              Any "tags" to place in the page header, such as <NxCode>NxTag</NxCode> or
+              <NxCode>NxPolicyViolationIndicator</NxCode>, should be wrapped by this element.
             </td>
           </tr>
         </tbody>
       </table>
     </GalleryDescriptionTile>
 
-    <GalleryExampleTile title="NX Page Title Example"
+    <GalleryExampleTile title="Basic page title"
                         id="nx-page-title-example"
                         defaultCheckeredBackground={true}
                         htmlExample={nxPageTitleCode}
                         codeExamples={nxPageTitleCode}>
       A simple example of an <code className="nx-code">nx-page-title</code>.
     </GalleryExampleTile>
 
-    <GalleryExampleTile title="NX Page Title with Actions Example"
+    <GalleryExampleTile title="Page title with actions"
                         id="nx-page-title-actions-example"
                         defaultCheckeredBackground={true}
                         htmlExample={nxPageTitleActionsCode}
                         codeExamples={nxPageTitleActionsCode}>
       An example of <code className="nx-code">nx-page-title</code> with actions in the form of a drop-down.
```

Comment:
It isn't a dropdown

Reply:
> It isn't a dropdown

Removed reference to dropdown here: https://github.com/sonatype/sonatype-react-shared-components/pull/363/commits/cfe9ef4c8fe3058d8ed0cef6ad2ab43643a303fb